src/settings.ts

```diff
@@ -1,5 +1,8 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 export const settings = {
+  sentry:{
+    dsn: process.env.REACT_APP_MAGIC_BOOST_DSN!
```

Comment:
remember to update example.env

You can copy the same DSN everywhere, that's not a secret